spyder/plugins/completion/languageserver/confpage.py

```diff
@@ -1029,6 +1044,7 @@ def setup_page(self):
 
         # --- Tabs organization ---
         self.tabs = QTabWidget()
+        self.tabs.addTab(self.create_tab(clients_widget), _('Clients'))
```

Comment:
Instead of adding a new tab just with these three options, perhaps we should add the new checkboxes to the "Advanced" one? I mean, we could create two QGroupBoxes: one for the advanced options of the PyLS and the other to select among the available completion clients.

If you think that's not a good suggestion, then we should put this tab to the left ot the "Advanced" tab.

@andfoy, any input on your side about this?

Reply:
>Instead of adding a new tab just with these three options, perhaps we should add the new checkboxes to the "Advanced" one? 

Sure